src/Components/CurrentWeather.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import mostlycloudy from "../img/weather-icons/mostlycloudy.svg";
+import "../App.css";
+class CurrentWeather extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <main className="currentWeather">
+        <img class="main-img" src={mostlycloudy} alt="mostly cloudy" />
+        <h3 class="title">overcast clouds</h3>
+        <section class="flex-container1">
+          <p class="temp">
+            <b>Temprature</b>&nbsp;&nbsp;&nbsp;10°to 11°C&nbsp;
```

Comment:
Use margin or padding in CSS to get the spacing you want, rather than using &nbsp;
This may mean you have to wrap your numbers in a tag, such as `span`, so that you can target them more easily with a CSS selector.